src/nucleus/tt.ml

```diff
@@ -6,6 +6,8 @@ type ('a,'b) constrain =
 
 type ('a, 'b) abstraction = (Name.ident * 'a) * 'b
 
+type bound = int
```

Comment:
If we directly carry bound variables from `Syntax` to `Tt` then this should be indicated with `type bound = Syntax.bound`.